client/public/runner.html

```diff
@@ -121,6 +121,7 @@
         script.textContent = state.js;
         document.body.appendChild(script);
 
+        dispatchEvent(new Event("DOMContentLoaded"));
```

Comment:
This should be fired on the document, not the window (technically even the load one below has a legacy retarget to doc). Don't forget to make it bubble then.